conf/homeserver.yaml

```diff
@@ -1394,8 +1394,8 @@ account_threepid_delegates:
 # If the room already exists, make certain it is a publicly joinable
 # room. The join rule of the room must be set to 'public'.
 #
-#auto_join_rooms:
-#  - "#example:example.com"
+auto_join_rooms:
+  - "__AUTO_JOIN_ROOMS__"
```

Comment:
Well, did you test if it works if works when you set nothing here ? And same ideally we should be able to mange here as list, but it's not mandatory for now.

Reply:
here, @thardev could contribute a `getter/setter` with some `sed`

Reply:
solved thanks to @rosbeef